README.md

```diff
@@ -19,13 +19,57 @@ Common requests are:
 
 And **thank you** for your contribution. It adds to the monitoring :heart:.
 
+## Maintained Areas
+
+Maintainers do their best to cover the diverse set of plugins and adjacent technologies that make using Sensu a joy.
+
+| Area                         | Major Links                                                  | Maintainers                                                                                                                                        | Needs Contributors?                                                                              |
+|:-----------------------------|:-------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------------------------------------------------------------------------------------|
+| ![Sensu Org][sensu_pic]      | [Plugins Org][plug_org]</br>[Extensions Org][ext_org]        | **[Matt (@mbbroberg - Lead)][11]**</br>[Ben (@majormoses)][9]</br>[Eric (@eheydrick)][10]</br>[Luis (@luisdavim)][12]</br>[Shane (@sstarcher)][13] | **Yes!** [Read about how!][volunteer]                                                            |
+| ![Ruby Plugin][ruby_pic]     | [Ruby Plugin Library][ruby_lib]                              | **[Ben (@majormoses - Lead)][9]**</br>[Eric (@eheydrick)][10]</br>[Cameron (@cwjohnston)][17]                                                      | **Yes!** Please [claim an open issue!](https://github.com/sensu-plugins/sensu-plugin/issues)     |
+| ![Python Plugin][python_pic] | [Python Plugin Library][py_lib]                              | **[Barry O'Rourke (@barryorourke - Lead)][14]**                                                                                                    | **Yes!** [Help wanted on testing 🙏][py_helpwanted]                                              |
+| ![Windows][windows_pic]      | [Windows Plugins][windows]</br>[SaltStack Role][saltstack]   |                                                                                                                                                    | **Yes!** [We're looking for a maintainer!](https://github.com/sensu-plugins/community/issues/74) |
+| ![Puppet][puppet_pic]        | [Puppet Module][puppet]                                      | **[Garrett (@ghoneycutt - Lead)](https://github.com/ghoneycutt)**                                                                                  | We have dedicated maintainers right now!                                                         |
+| ![Chef][chef_pic]            | [Chef Cookbook][chef]                                        | **[Ben (@majormoses - Lead)][9]**                                                                                                                  | **Yes!** Please [volunteer to help out.](https://github.com/sensu-plugins/community/issues/75)   |
```

Comment:
Cameron is the lead on this, I help maintain it but I don't have full access. I can merge, push to marketplace, etc but do not have access to repo settings and such.

Reply:
✅